### ISSUE TITLE
cpan file lists WWW:: 

### ISSUE THREAD

**Issue by guest20** [genio/digest-bcrypt#2]
The cpanfile lists:

```
    requires 'WWW::Mechanize';
    requires 'WWW::Shorten' => '3.09';
```

The only place these appear is in the depends declarations:
    https://github.com/genio/digest-bcrypt/search?utf8=%E2%9C%93&q=WWW

maybe they could be removed.

**Comment by genio** [comment]
Gah!  I really shouldn't copy-paste from my other modules sometimes as I apparently don't clean everything up.

Thanks for the report.

Addressed in: f3226e2

**Comment by genio** [comment]
Thanks again for the report!

1.206 has been pushed with the fix.

https://metacpan.org/release/CAPOEIRAB/Digest-Bcrypt-1.206
